### ISSUE TITLE
Bring up remaining AXPlatformNodeTextRangeProviderWin unittests

### ISSUE THREAD

**Issue by yaakovschectman** [flutter/flutter#117012]
Currently, as a result of adding implementations for `TextRangeProvider` and `TextProvider` for [flutter/flutter#11629](https://github.com/flutter/flutter/issues/116219), a few of the unit tests for `AXPlatformNodeTextRangeProviderWin`, as modeled after the Chromium codebase, are left disabled for now, as they refer to or rely on functionality that we lack or do not use, such as certain attribute values or ignored state.
